homeassistant/components/ios/manifest.json

```diff
@@ -3,6 +3,6 @@
   "name": "Apple iOS",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/ios",
-  "dependencies": ["device_tracker", "http", "zeroconf"],
+  "dependencies": ["device_tracker", "http", "zeroconf", "notify"],
```

Comment:
If we load the platform with `load_platform`, we shouldn't need to depend on the integration here.

Reply:
The problem is that load_platform throws an exception which i tried to fix, apparently in a wrong way :)
the assertion it throws is that it gets passed a null config and fails the assertion ("you need to pass the real hass config"), so during tests, "notify" fails to load, but not sure what is different when running live. Any suggestions?

Reply:
The fifth argument to `discovery.load_platform` should be the original config yaml. We need to make sure it's passed where we call this function.

Reply:
Yeah it's not allowed to be empty. `{ "homeassistant: {} }` will do.

Reply:
I didn't find any place that it wasn't passed correctly. The only problem is that in a test hass, config is set to {}. perhaps we start the test home assistant with some {foo:bar} so it is not empty since hass config cannot really be empty (at least defaultconfig:, right?)

Reply:
See my comment below